### PR TITLE
Feature/notifications

### PR DESCRIPTION
**Pull request by Mykyta-Pysarev**
Replace Mocks with real Notifications (only likes for now).
Made some minor adjustments  to tweet, so you can click on it and reach single tweet with comments.

image

### CODE REVIEW

File: src/components/SingleNotification.tsx [hunk 1]
```diff
@@ -51,26 +77,40 @@ const SingleNotification = ({
...
+                      className="me-2"
+                      roundedCircle
+                      fluid
+                      src={
+                        item.user?.profile?.avatar
+                          ? item.user.profile.avatar
+                          : "https://abs.twimg.com/sticky/default_profile_images/default_profile_400x400.png"
+                      }
+                      key={item._id}
+                    />
```
Comment (DavidKorystov) [inline src/components/SingleNotification.tsx]: Replace StyledImage on Avatar.tsx component

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

File: src/components/SingleNotification.tsx (hunk 1 above)
Comment (Mykyta-Pysarev) [reply src/components/SingleNotification.tsx]: Thanks for the tip. Done.
image

**Review by VitaliiDenysiuk — APPROVED** [review]
lgtm

File: src/containers/NotificationsPage.tsx [hunk 2]
```diff
@@ -1,26 +1,34 @@
-import SingleNotification from "../components/SingleNotification";
-import { notificationPageMocks } from "../MOCKS/notificationsPage";
```
Comment (Barylo) [inline src/containers/NotificationsPage.tsx:2]: Delete notificationPage.js from MOCKS

**Review by DavidKorystov — APPROVED** [review]

File: src/containers/NotificationsPage.tsx (hunk 2 above)
Comment (Mykyta-Pysarev) [reply src/containers/NotificationsPage.tsx:2]: Done.

**Review by Barylo — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**